.install/alpine_linux_3.sh

```diff
@@ -6,6 +6,8 @@ $MODE apk update
 
 $MODE apk add --no-cache build-base gcc g++ make linux-headers openblas-dev \
     xsimd curl wget git python3 python3-dev py3-pip openssl openssl-dev \
-    tar xz which rsync bsd-compat-headers clang clang17-libclang curl
+    tar xz which rsync bsd-compat-headers curl \
+    bash cargo gcc autoconf automake py3-cryptography \
+    musl-dev libffi-dev openssh py-virtualenv clang18-libclang
```

Comment:
For JSON build? Does it affect CI duration?

Reply:
I tried adding the installation of JSON module when the build failed, but these changes did not effect the JSON build for Alpine, so it can be reverted